app/models/sww/event.rb

```diff
@@ -11,5 +11,9 @@ module Sww::Event
   included do
     self.used_attributes += [:waiting_list]
     self.supports_applications = true
+
+    def group_names
```

Comment:
Should do the same as overwriting the partial, event.group_names seems to only be used for run_by in events and public events